0.X/docker-entrypoint.sh

```diff
@@ -10,31 +10,35 @@ set -e
 # You can set CONSUL_BIND_INTERFACE to the name of the interface you'd like to
 # bind to and this will look up the IP and pass the proper -bind= option along
 # to Consul.
-CONSUL_BIND=
-if [ -n "$CONSUL_BIND_INTERFACE" ]; then
-  CONSUL_BIND_ADDRESS=$(ip -o -4 addr list $CONSUL_BIND_INTERFACE | head -n1 | awk '{print $4}' | cut -d/ -f1)
-  if [ -z "$CONSUL_BIND_ADDRESS" ]; then
-    echo "Could not find IP for interface '$CONSUL_BIND_INTERFACE', exiting"
-    exit 1
-  fi
+if [ -z "$CONSUL_BIND" ]; then
```

Comment:
since this is checking for an empty value, it's probably safe to remove the value being reset on the following line, same for the other instances of `CONSUL_CLIENT`

Reply:
Fixed